command/scaling_policy_info_test.go

```diff
@@ -35,19 +35,19 @@ func TestScalingPolicyInfoCommand_Run(t *testing.T) {
 	cmd := &ScalingPolicyInfoCommand{Meta: Meta{Ui: ui}}
 
 	// Calling without the policyID should result in an error.
-	if code := cmd.Run([]string{"-address=" + url}); code != 1 {
+	if code := cmd.Run([]string{"-address=" + url, "first", "second"}); code != 1 {
```

Comment:
This test is now calling `nomad scaling policy info` with 2 arguments. 

Maybe keep the previous test (no argument) and add this one? The output should be the same for both.

Reply:
I can't comment on unchanged lines, so I will pile it up here 😅 

There could be a new test for prefix match similar to this one:

```go
	if code := cmd.Run([]string{"-address=" + url, policies[0].ID}); code != 0 {
		t.Fatalf("expected cmd run exit code 0, got: %d", code)
	}
```

Where the argument is a substring of the ID (and maybe also one for a single character ID).